content/docs/command-reference/plot/index.md

```diff
@@ -0,0 +1,239 @@
+# plot
+
+Contains commands to visualize
+[continuous metrics](/doc/command-reference/plot#continuous-metrics) in
+structured files (JSON, CSV, or TSV): [show](/doc/command-reference/plot/show),
+[diff](/doc/command-reference/plot/diff).
+
+## Synopsis
+
+```usage
+usage: dvc plot [-h] [-q | -v] {show,diff} ...
+
+positional arguments:
+  COMMAND
+    show         Generate a plot image file from a continuous metrics file.
+    diff         Plot continuous metrics differences between commits in the DVC
+                 repository, or between the last commit and the workspace.
+```
+
+## Description
+
+DVC provides a set of commands to visualize _continuous metrics_ of machine
+learning experiments. Usual examples of plots are AUC curves, loss functions,
+and confusion matrices.
+
+Continuous metrics represent plots, and should be stored as data series in one
+of the supported [file formats](#file-formats). These files are usually created
+by users or generated by user modeling or data processing code.
+
+The plot commands can work with these continuous metrics files that are commited
+to a repository history, data files controlled by DVC or files from workspace.
+For examlpe, the command `dvc plot diff` generates a plot with two versions of
+the metrics:
+
+```dvc
+$ dvc plot diff -d logs.csv
+file:///Users/dmitry/src/plot/logs.html
+```
+
+![](/img/plot_auc.svg)
+
+### Difference between continuous and scolar metrics
```

Comment:
scalar

Reply:
Adding suggestion for quick commit from GH:
```suggestion
### Difference between continuous and scalar metrics
```